lib/bump.rb

```diff
@@ -19,13 +19,14 @@ class << self
 
       def defaults
         {
+          :tag => ::Bump.tag_by_default,
```

Comment:
a warning here for "order is the order of rake arguments" would be nice

Reply:
Do you think [:tag, :commit, :bundle] order is okay? I kept :tag at first so that current `rake bump:patch[true]` is respected.
The task will accept the arguments in the same order as in `defaults`.

Reply:
oder seems alright, it's kinda unintuitive/magic whatever way they are used ... env vars would be better or having the Rakefile set the defaults as necessary for the project